bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/v18/V18AssortedRpc.scala

```diff
@@ -0,0 +1,38 @@
+package org.bitcoins.rpc.client.v18
+import org.bitcoins.rpc.client.common.Client
+import org.bitcoins.rpc.jsonmodels.{
+  DeriveAddressesResult,
+  GetDescriptorInfoResult,
+  GetNodeAddressesResult,
+  ListWalletDirResult
+}
+import org.bitcoins.rpc.serializers.JsonSerializers._
+import play.api.libs.json._
+
+import scala.concurrent.Future
+
+trait V18AssortedRpc {
+  self: Client =>
+
+  def getNodeAddresses(
+      count: Option[Double]): Future[GetNodeAddressesResult] = {
```

Comment:
The number here is an `Int`, not a double. Also, this would make for a better user experience: 


```scala
private def getNodeAddresses(
      count: Option[Int]): Future[GetNodeAddressesResult] = {
     bitcoindCall[GetNodeAddressesResult]("getNodeAddresses",
                                          List(Json.toJson(count)))
   }

def getNodeAddresses(count: Int): Future[GetNodeAddressesResult] = getNodeAddresses(Some(count))
def getNodeAddresses(): Future[GetNodeAddressesResult] = getNodeAddresses(None)
```

That way the user can easily call the method both with and without a `count` argument